packages/cozy-clisk/src/launcher/saveFiles.js

```diff
@@ -7,17 +7,27 @@ import retry from 'bluebird-retry'
 const log = Minilog('saveFiles')
```

Comment:
This is not the right semantic see https://github.com/cozy/cozy-guidelines/#breaking-change 🙏

Reply:
Now it should be better

Reply:
Nop. No space before the `:`

Reply:
This time is the right one